tool-plugins/nuxt-starter/app.config.ts

```diff
@@ -0,0 +1,8 @@
+export default defineAppConfig({
+	type: 'space-plugin',
```

Comment:
this is tool-plugin right?

Reply:
Sorry I wasn't clear! #77 and #78 do the actual jobs, so you can review them. They fix all the things in this PR.

Reply:
Oh sorry, I misunderstood. I reviewed both already so I can also approve this one.